h2o-automl/build.gradle

```diff
@@ -4,6 +4,7 @@ dependencies {
   compile project(":h2o-genmodel")
   compile project(":h2o-core")
   compile project(":h2o-algos")
+  compile project(":h2o-ext-xgboost")
```

Comment:
I don't think this is an issue for H2O deployments but I wanted to point out that xgboost is designed as an optional extension. It is bundled in H2O, h2odriver and SW but users might choose not to include xgboost in their custom H2O builds.
It would be better if xgboost was loaded dynamically based on availability but I do understand that would be a bigger change and this is fine for now.

Reply:
I just need XGBoostParameters class when training XGBoost model and no class from XGBoost extension is loaded before that, indeed there XGBoost extension is first detected using ExtensionManager and no XGBoost-related code should be executed if the extension is missing.
Having XGBoostParameters in the imports should be irrelevant for the classloader, so I think we're fine already. Need to double-check though.